Description: Default config
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-26

Index: glance/etc/glance-api.conf
===================================================================
--- glance.orig/etc/glance-api.conf
+++ glance/etc/glance-api.conf
@@ -1683,7 +1683,7 @@ connection = sqlite:////var/lib/glance/g
 # service name removed. For example, if your paste section name is
 # [pipeline:glance-api-keystone] use the value "keystone" (string
 # value)
-#flavor = <None>
+flavor = caching
 
 # Name of the paste configuration file. (string value)
 #config_file = <None>
Index: glance/etc/glance-registry.conf
===================================================================
--- glance.orig/etc/glance-registry.conf
+++ glance/etc/glance-registry.conf
@@ -1390,7 +1390,7 @@ connection = sqlite:////var/lib/glance/g
 # service name removed. For example, if your paste section name is
 # [pipeline:glance-api-keystone] use the value "keystone" (string
 # value)
-#flavor = <None>
+flavor = caching
 
 # Name of the paste configuration file. (string value)
 #config_file = <None>
