Description: Fixes glance-api.conf and glance-registry.conf
Author: Ghe Rivero
Forwarded: no

Index: glance/etc/glance-api.conf
===================================================================
--- glance.orig/etc/glance-api.conf	2014-02-14 11:22:07.000000000 +0800
+++ glance/etc/glance-api.conf	2014-02-14 11:22:09.000000000 +0800
@@ -442,4 +442,4 @@
 # service name removed. For example, if your paste section name is
 # [pipeline:glance-api-keystone], you would configure the flavor below
 # as 'keystone'.
-#flavor=
+flavor = caching
Index: glance/etc/glance-registry.conf
===================================================================
--- glance.orig/etc/glance-registry.conf	2014-02-14 11:22:07.000000000 +0800
+++ glance/etc/glance-registry.conf	2014-02-14 11:22:09.000000000 +0800
@@ -96,4 +96,4 @@
 # service name removed. For example, if your paste section name is
 # [pipeline:glance-registry-keystone], you would configure the flavor below
 # as 'keystone'.
-#flavor=
+flavor = caching
