Description: Fixes glance-api.conf and glance-registry.conf
Author: Ghe Rivero
Forwarded: no

--- a/etc/glance-api.conf
+++ b/etc/glance-api.conf
@@ -282,6 +282,9 @@
 # RADOS user to authenticate as (only applicable if using cephx)
 rbd_store_user = glance
 
+
+[paste_deploy]
+flavor = caching
 # RADOS pool in which images are stored
 rbd_store_pool = images
 
--- a/etc/glance-registry.conf
+++ b/etc/glance-registry.conf
@@ -67,6 +67,10 @@
 # CA certificate file to use to verify connecting clients
 #ca_file = /path/to/cafile
 
+[paste_deploy]
+flavor = caching
+
+
 [keystone_authtoken]
 auth_host = 127.0.0.1
 auth_port = 35357
