--- a/etc/glance-api.conf
+++ b/etc/glance-api.conf
@@ -240,3 +240,6 @@
 
 # Base directory that the Image Cache uses
 image_cache_dir = /var/lib/glance/image-cache/
+
+[paste_deploy]
+flavor = caching
\ No newline at end of file
--- a/etc/glance-registry.conf
+++ b/etc/glance-registry.conf
@@ -59,3 +59,6 @@
 
 # Private key file to use when starting registry server securely
 # key_file = /path/to/keyfile
+
+[paste_deploy]
+flavor = caching
\ No newline at end of file
