Description: Fixes glance-api.conf and glance-registry.conf
Author: Ghe Rivero
Forwarded: no

Index: glance/etc/glance-api.conf
===================================================================
--- glance.orig/etc/glance-api.conf	2012-09-10 15:37:20.000000000 +0200
+++ glance/etc/glance-api.conf	2012-09-10 18:42:20.082309794 +0200
@@ -282,3 +282,6 @@
 
 # Base directory that the Image Cache uses
 image_cache_dir = /var/lib/glance/image-cache/
+
+[paste_deploy]
+flavor = caching
\ No newline at end of file
Index: glance/etc/glance-registry.conf
===================================================================
--- glance.orig/etc/glance-registry.conf	2012-09-10 15:37:20.000000000 +0200
+++ glance/etc/glance-registry.conf	2012-09-10 18:43:31.210312746 +0200
@@ -66,3 +66,7 @@
 
 # CA certificate file to use to verify connecting clients
 # ca_file = /path/to/cafile
+
+[paste_deploy]
+flavor = caching
+
