Description: Default config
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-26

--- glance-11.0.0~rc1.orig/etc/glance-api.conf
+++ glance-11.0.0~rc1/etc/glance-api.conf
@@ -1302,7 +1302,7 @@ connection = sqlite:////var/lib/glance/g
 # service name removed. For example, if your paste section name is
 # [pipeline:glance-api-keystone] use the value "keystone" (string
 # value)
-#flavor = <None>
+flavor = caching
 
 # Name of the paste configuration file. (string value)
 #config_file = <None>
--- glance-11.0.0~rc1.orig/etc/glance-registry.conf
+++ glance-11.0.0~rc1/etc/glance-registry.conf
@@ -1073,7 +1073,7 @@ connection = sqlite:////var/lib/glance/g
 # service name removed. For example, if your paste section name is
 # [pipeline:glance-api-keystone] use the value "keystone" (string
 # value)
-#flavor = <None>
+flavor = caching
 
 # Name of the paste configuration file. (string value)
 #config_file = <None>
