Description: don't rely on network
Author: Mehdi Abaakouk <sileht@sileht.net>

--- a/glance/openstack/common/setup.py
+++ b/glance/openstack/common/setup.py
@@ -259,7 +259,7 @@
             # sdist.sdist is an old style class, can't use super()
             sdist.sdist.run(self)
 
-    cmdclass['sdist'] = LocalSDist
+    #cmdclass['sdist'] = LocalSDist
 
     # If Sphinx is installed on the box running setup.py,
     # enable setup.py to build the documentation, otherwise,
@@ -329,7 +329,7 @@
 def get_pre_version(projectname, base_version):
     """Return a version which is leading up to a version that will
        be released in the future."""
-    if os.path.isdir('.git'):
+    if False and os.path.isdir('.git'):
         current_tag = _get_git_current_tag()
         if current_tag is not None:
             version = current_tag
@@ -351,7 +351,7 @@
     revision if there is one, or tag plus number of additional revisions
     if the current revision has no tag."""
 
-    if os.path.isdir('.git'):
+    if False and os.path.isdir('.git'):
         version = _get_git_post_version()
         write_versioninfo(projectname, version)
         return version
