Description: Fixe most pep8 errors.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed

--- a/glance/common/utils.py
+++ b/glance/common/utils.py
@@ -386,7 +386,7 @@
 
     height_width = func.get(platform.os.name, _get_terminal_size_unknownOS)()
 
-    if height_width == None:
+    if height_width is None:
         raise exception.Invalid()
 
     for i in height_width:
--- a/glance/db/sqlalchemy/migrate_repo/versions/003_add_disk_format.py
+++ b/glance/db/sqlalchemy/migrate_repo/versions/003_add_disk_format.py
@@ -93,8 +93,8 @@
         images.outerjoin(image_properties,
                          and_(images.c.id == image_properties.c.image_id,
                               image_properties.c.key == 'type'))]).where(
-            and_(image_properties.c.image_id == None,
-                 images.c.type != None))
+            and_(image_properties.c.image_id is None,
+                 images.c.type is not None))
     image_records = conn.execute(sel).fetchall()
     property_insert = image_properties.insert()
     for record in image_records:
--- a/glance/db/sqlalchemy/migrate_repo/versions/010_default_update_at.py
+++ b/glance/db/sqlalchemy/migrate_repo/versions/010_default_update_at.py
@@ -64,7 +64,7 @@
     conn = migrate_engine.connect()
     conn.execute(
         images_table.update(
-            images_table.c.updated_at == None,
+            images_table.c.updated_at is None,
             {images_table.c.updated_at: images_table.c.created_at}))
 
 
