Description: Fixe most pep8 errors.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed

--- glance-2012.1.orig/glance/tests/stubs.py
+++ glance-2012.1/glance/tests/stubs.py
@@ -131,7 +131,7 @@ def stub_out_registry_and_store_server(s
 
         def endheaders(self):
             hl = [i.lower() for i in self.req.headers.keys()]
-            assert not ('content-length' in hl and \
+            assert not ('content-length' in hl and
                         'transfer-encoding' in hl), \
                 'Content-Length and Transfer-Encoding are mutally exclusive'
 
--- glance-2012.1.orig/glance/tests/unit/test_cfg.py
+++ glance-2012.1/glance/tests/unit/test_cfg.py
@@ -750,7 +750,7 @@ class OptDumpingTestCase(BaseTestCase):
         self.assertEquals(logger.logged, [
                 "*" * 80,
                 "Configuration options gathered from:",
-                "command line args: ['--foo', 'this', '--blaa-bar', 'that', "\
+                "command line args: ['--foo', 'this', '--blaa-bar', 'that', "
                 "'--blaa-key', 'admin', '--passwd', 'hush']",
                 "config files: []",
                 "=" * 80,
--- glance-2012.1.orig/glance/common/client.py
+++ glance-2012.1/glance/common/client.py
@@ -572,9 +572,9 @@ class BaseClient(object):
             return (e.errno != errno.ESPIPE)
 
     def _sendable(self, body):
-        return (SENDFILE_SUPPORTED      and
+        return (SENDFILE_SUPPORTED and
                 hasattr(body, 'fileno') and
-                self._seekable(body)    and
+                self._seekable(body) and
                 not self.use_ssl)
 
     def _iterable(self, body):
--- glance-2012.1.orig/glance/common/utils.py
+++ glance-2012.1/glance/common/utils.py
@@ -341,7 +341,7 @@ def get_terminal_size():
 
     height_width = func.get(platform.os.name, _get_terminal_size_unknownOS)()
 
-    if height_width == None:
+    if height_width is None:
         raise exception.Invalid()
 
     for i in height_width:
--- glance-2012.1.orig/glance/common/cfg.py
+++ glance-2012.1/glance/common/cfg.py
@@ -748,7 +748,7 @@ class ConfigOpts(object):
                                               usage=self.usage)
         self._cparser = None
 
-        self.register_cli_opt(\
+        self.register_cli_opt(
             MultiStrOpt('config-file',
                         default=self.default_config_files,
                         metavar='PATH',
@@ -838,7 +838,7 @@ class ConfigOpts(object):
         :return: False if the opt was already register, True otherwise
         :raises: DuplicateOptError, ArgsAlreadyParsedError
         """
-        if self._args != None:
+        if self._args is not None:
             raise ArgsAlreadyParsedError("cannot register CLI option")
 
         if not self.register_opt(opt, group):
--- glance-2012.1.orig/glance/common/auth.py
+++ glance-2012.1/glance/common/auth.py
@@ -133,7 +133,7 @@ class KeystoneStrategy(BaseStrategy):
                 # 3. In some configurations nova makes redirection to
                 # v2.0 keystone endpoint. Also, new location does not
                 # contain real endpoint, only hostname and port.
-                if  'v2.0' not in auth_url:
+                if 'v2.0' not in auth_url:
                     auth_url = urlparse.urljoin(auth_url, 'v2.0/')
             else:
                 # If we sucessfully auth'd, then memorize the correct auth_url
