Description: Fixes default sql_connection in glance-registry.conf
Author: Julien Danjou
Forwarded: no

Index: glance/etc/glance-registry.conf
===================================================================
--- glance.orig/etc/glance-registry.conf	2012-09-10 18:43:31.210312746 +0200
+++ glance/etc/glance-registry.conf	2012-09-10 18:44:00.126313942 +0200
@@ -25,7 +25,7 @@
 # SQLAlchemy connection string for the reference implementation
 # registry server. Any valid SQLAlchemy connection string is fine.
 # See: http://www.sqlalchemy.org/docs/05/reference/sqlalchemy/connections.html#sqlalchemy.create_engine
-sql_connection = sqlite:///glance.sqlite
+sql_connection = sqlite:////var/lib/glance/glance.sqlite
 
 # Period in seconds after which SQLAlchemy should reestablish its connection
 # to the database.
