--- a/glance/tests/functional/test_api.py
+++ b/glance/tests/functional/test_api.py
@@ -25,7 +25,7 @@
 
 from glance.common import utils
 from glance.tests import functional
-from glance.tests.utils import execute, skip_if_disabled, minimal_headers
+from glance.tests.utils import execute, skip_if_disabled, minimal_headers, skip_test
 
 FIVE_KB = 5 * 1024
 FIVE_GB = 5 * 1024 * 1024 * 1024
@@ -1299,7 +1299,7 @@
 
         self.stop_servers()
 
-    @skip_if_disabled
+    @skip_test
     def test_unsupported_default_store(self):
         """
         We test that a mis-configured default_store causes the API server
